beetsplug/convert.py

```diff
@@ -175,8 +178,8 @@ def encode(self, command, source, dest, pretend=False):
         args = shlex.split(command)
         for i, arg in enumerate(args):
             args[i] = Template(arg).safe_substitute({
-                'source': source,
-                'dest': dest,
+                'source': source.decode('utf8'),
+                'dest': dest.decode('utf8'),
```

Comment:
Note to self: Need to come back and test this. I would have expected we were constructing bytestrings for the command. Does a unicode command work?